test/validate/ferc1_test.py

```diff
@@ -73,52 +73,6 @@ def test_no_null_cols_ferc1(live_dbs, asset_value_loader, cols, asset_key):
     )
 
 
-@pytest.mark.parametrize(
-    "asset_key,unique_subset",
-    [
-        (
-            "_out_ferc1__yearly_plants_utilities",
-            ["utility_id_ferc1", "plant_name_ferc1"],
-        ),
-        (
-            "out_ferc1__yearly_steam_plants_fuel_by_plant_sched402",
-            ["report_year", "utility_id_ferc1", "plant_name_ferc1"],
-        ),
-        (
-            "out_ferc1__yearly_hydroelectric_plants_sched406",
-            [
-                "report_year",
-                "utility_id_ferc1",
-                "plant_name_ferc1",
-                "capacity_mw",  # Why does having capacity here make sense???
-            ],
-        ),
-        (
-            "out_ferc1__yearly_pumped_storage_plants_sched408",
-            [
-                "report_year",
-                "utility_id_ferc1",
-                "plant_name_ferc1",
-                "capacity_mw",  # Why does having capacity here make sense???
-            ],
-        ),
```

Comment:
Inclusion of `capacity_mw` here makes these kinda fishy -- I think it's probably a vestige of an older version of these tables? Not entirely sure. Current PK for these tables is a pseudo key `record_id` that just says where the data came from in the FERC 1 mess.

It might be reasonable to turn each of these into non-PK multi-column uniqueness tests within the `dbt` schemas.

Reply:
I went ahead and added them, since the test has been passing and it seems like a real, though heuristic, expectation.